yt/utilities/answer_testing/framework.py

```diff
@@ -214,7 +214,7 @@ def get(self, ds_name, default=None):
         except urllib.error.HTTPError:
             raise YTNoOldAnswer(url)
         else:
-            for this_try in range(3):
+            for _ in range(3):
```

Comment:
this loop feels weird to me but it may be correct (try up to 3 times the same exact request with a 0.01s delay between each attempt)

Reply:
(in a separate PR)

Alternately, we should just clean up this whole file once pytest goes in.